lib/internal/bootstrap_node.js

```diff
@@ -249,6 +250,21 @@
     });
   }
 
+  function setupInspectorAPI() {
+    var addCommandLineAPIMethod = process.inspector.addCommandLineAPIMethod;
```

Comment:
Can we remove this method from inspector object right here to be sure that it won't leak?